deploy/templates/components/manager/manager.yaml

```
@@ -89,7 +89,7 @@ spec:
             - name: RELATED_IMAGE_devworkspace_webhook_server
               value: "quay.io/devfile/devworkspace-controller:next"
             - name: RELATED_IMAGE_pvc_cleanup_job
-              value: "quay.io/libpod/busybox:1.30.1"
+              value: "registry.access.redhat.com/ubi8/ubi-minimal:8.4"
```

```suggestion
              value: "registry.access.redhat.com/ubi8/ubi-minimal:8.4-205.1626828526"
```
the best practice is to avoid using moving tags.